ui/src/app/groupings/[groupingPath]/layout.tsx

```diff
@@ -2,19 +2,31 @@ import ExportDropdown from './_components/export-dropdown';
 import GroupingHeader from './_components/grouping-header';
 import ReturnButtons from './_components/return-buttons';
 import SideNav from './_components/side-nav';
-import { groupingDescription } from '@/lib/fetchers';
+import { groupingDescription, allGroupingMembers } from '@/lib/fetchers';
 
 const GroupingPathLayout = async ({ params, tab }: { params: { groupingPath: string }; tab: React.ReactNode }) => {
     const groupPath = decodeURIComponent(params.groupingPath);
     const { description } = await groupingDescription(groupPath);
     const groupName = groupPath.split(':').pop() as string;
     const fromManageSubject = groupPath.includes('manage-person');
 
+    const groupPaths = [`${groupPath}:include`, `${groupPath}:exclude`, `${groupPath}:basis`, `${groupPath}:owners`];
+    const sortString = 'name';
+    const isAscending = true;
+    const fetchGroupingMembers = await allGroupingMembers(groupPaths, sortString, isAscending);
```

Comment:
I think we should avoid calling allGroupingMembers in this layout component. This can increase load times for large groupings because we have to wait for allGroupingMembers to fetch. Instead, use a server action. Feel free to send me a dm if you need more info.